src/main/java/gov/cabinetoffice/gap/adminbackend/services/GrantMandatoryQuestionService.java

```diff
@@ -50,22 +89,19 @@ public ByteArrayOutputStream exportSpotlightChecks(Integer schemeId) {
                     + " is unable to access mandatory questions with scheme id " + schemeId);
         }
 
-        final List<GrantMandatoryQuestions> grantMandatoryQuestions = getGrantMandatoryQuestionBySchemeAndCompletedStatus(
-                schemeId);
         log.info("Found {} mandatory questions in COMPLETED state for scheme ID {}", grantMandatoryQuestions.size(),
                 schemeId);
 
         List<List<String>> exportData = new ArrayList<>();
         grantMandatoryQuestions.forEach(grantMandatoryQuestion -> {
             try {
-                exportData.add(buildSingleSpotlightRow(grantMandatoryQuestion));
+                exportData.add(buildSingleSpotlightRow(grantMandatoryQuestion, addOrgType));
             }
             catch (SpotlightExportException e) {
                 log.error("Problem extracting data: " + e.getMessage());
             }
         });
-
-        return XlsxGenerator.createResource(SpotlightHeaders.SPOTLIGHT_HEADERS, exportData);
+        return exportData;
```

Comment:
Could simplify this too, unless its desired that we should continue despite errors creating rows?  
```
   return grantMandatoryQuestions.stream()
                .map(grantMandatoryQuestion -> buildSingleSpotlightRow(grantMandatoryQuestion, addOrgType))
                .toList();
```